src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1194,6 +1195,57 @@ IF(PLUS_USE_CAPISTRANO_VIDEO)
 ENDIF()
 
 # --------------------------------------------------------------------------
+# WinProbe support
+IF(PLUS_USE_WINPROBE_VIDEO)
+  SET(WinProbe_SRCS
+    WinProbe/vtkPlusWinProbeVideoSource.cxx
+    )
+  IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+    SET(WinProbe_HDRS
+      WinProbe/vtkPlusWinProbeVideoSource.h
+      )
+  ENDIF()
+  
+  SET(PlusDataCollection_HDRS ${PlusDataCollection_HDRS}
+    ${WinProbe_HDRS}
+    )
+
+  SET(PlusDataCollection_SRCS ${PlusDataCollection_SRCS}
+    ${WinProbe_SRCS}
+    )
+
+  SET(PlusDataCollection_INCLUDE_DIRS ${PlusDataCollection_INCLUDE_DIRS}     
+    ${WINPROBESDK_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}/WinProbe
+    )
+
+  FILE(GLOB WinProbeXMLFiles "${WINPROBESDK_DIR}/xml/*.xml")
+  FILE(COPY ${WinProbeXMLFiles} DESTINATION ${CMAKE_BINARY_DIR})
```

Comment:
Highly recommend using PLUS_EXECUTABLE_OUTPUT_DIR (I think, double check the exact variable name)